x/sync/README.md

```diff
@@ -1,3 +1,59 @@
+# `sync` package
+
+## Overview
+
+This package implements a client and server that allows for the syncing of a [MerkleDB](../merkledb/README.md). The servers have an up to date version of the database, and the clients have an out of date version of the database or an empty database.
+
+It's planned that these client and server implementations will eventually be compatible with Firewood.
+
+## Messages
+
+There are four message types sent between the client and server:
+
+1. `SyncGetRangeProofRequest`
+2. `RangeProof`
+3. `SyncGetChangeProofRequest`
+4. `SyncGetChangeProofResponse`
```

Comment:
I hate how much `RangeProof` sticks out here

Reply:
Would you hate it if we made a response struct that just had the range proof in it?

Reply:
We could rename it in the proto if you want, but if we do let's do it in a separate PR